symbiosis/resource_cluster.go

```diff
@@ -41,20 +41,12 @@ func ResourceCluster() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"configuration": {
-				Type:     schema.TypeSet,
-				ForceNew: true,
-				Optional: true,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"enable_nginx_ingress": {
-							Type:     schema.TypeBool,
-							Default:  false,
-							ForceNew: true,
-							Optional: true,
-						},
-					},
-				},
+			"is_highly_available": {
+				Type:        schema.TypeBool,
+				ForceNew:    true,
+				Optional:    true,
+				Default:     false,
+				Description: "When set to true, it will deploy a highly available control plane.",
```

Comment:
Perhaps this is clearer? 
```suggestion
				Description: "When set to true it will deploy a highly available control plane with multiple replicas for redundancy.",
```